xep.dtd

```diff
@@ -121,7 +121,7 @@ THE SOFTWARE.
   alt CDATA ''
   height CDATA ''
   width CDATA ''>
-<!ELEMENT link (#PCDATA)* >
+<!ELEMENT link (#PCDATA | cite)* >
```

Comment:
I don't fully understand what the consequences of this are; we probably shouldn't start adding random things to link though. Remember it gets used in PDFs, HTML, etc.